int32 robot_id
---
RobotPosGoal response